tests/resources/MLTdata-preproduction.ecsv

```diff
@@ -1,209 +1,214 @@
 # %ECSV 1.0
 # ---
 # datatype:
-# - {name: mirror_panel_id, datatype: string, description: mirror panel ID}
-# - {name: mirror_panel_radius, unit: m, datatype: float32, description: mirror panel radius (2f)}
-# - {name: psf_opt, unit: 'mm', datatype: float32, description: Average spot size of MLT mirror panels from 2f measurement (d80), best value (not at nominal distance)}
-# - {name: psf, unit: 'mm', datatype: float32, description: Average spot size of MLT mirror panels from 2f measurement (d80) at nominal distance of 32.14 m}
+# - {name: mirror_id, datatype: string, description: mirror panel ID}
+# - {name: mirror_diameter, unit: m, datatype: float32, description: mirror panel radius (2f)}
```

Comment:
I don't know why it says 2f here and also it's not the radius it's the diameter.